apps/mobile-wallet/src/features/modals/ModalContent.tsx

```diff
@@ -38,6 +38,8 @@ export interface ModalFlatListContentProps<T> extends ModalContentBaseProps, Fla
 
 const scrollDefaultProps = { bounces: false, scrollEventThrottle: 16 }
 
+// TODO: DELETE THIS COMPONENT ONCE BOTTOM MODAL IS REFACTORED
```

Comment:
We will actually keep ModalContent probably, but the file can be cleaned up a lot.